diff -Naur a/EXA/src/vivante_fbdev/vivante_fbdev_driver.c b/EXA/src/vivante_fbdev/vivante_fbdev_driver.c
--- a/EXA/src/vivante_fbdev/vivante_fbdev_driver.c	2014-04-02 14:37:35.000000000 +0200
+++ b/EXA/src/vivante_fbdev/vivante_fbdev_driver.c	2014-06-07 16:28:44.650000501 +0200
@@ -118,7 +118,7 @@
 
 static Bool noVIVExtension;
 
-static ExtensionModule VIVExt =
+static ExtensionModule VIVExts[] = {
 {
 	VIVExtensionInit,
 	VIVEXTNAME,
@@ -128,6 +128,7 @@
 	NULL,
 	NULL
 #endif
+}
 };
 
 Bool vivEnableCacheMemory = TRUE;
@@ -261,7 +262,7 @@
         setupDone = TRUE;
         xf86AddDriver(&FBDEV, module, HaveDriverFuncs);
         if(gVivFb)
-            LoadExtension(&VIVExt, FALSE);
+            LoadExtensionList(VIVExts, ARRAY_SIZE(VIVExts), FALSE);
 		return (pointer)1;
     } else {
         if (errmaj) *errmaj = LDR_ONCEONLY;
